Stripe/UI/UIImage+Stripe.m

```diff
@@ -0,0 +1,44 @@
+//
+//  UIImage+Stripe.m
+//  Stripe
+//
+//  Created by Ben Guo on 1/4/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import "UIImage+Stripe.h"
+#import "STPPaymentCardTextFieldViewModel.h"
+
+@implementation UIImage (Stripe)
+
++ (UIImage *)stp_amexCardImage {
```

Comment:
I think it'd be better to switch this around - in other words, these methods should just be calling `imageNamed:`, and then STPPaymentCardTextFieldViewModel would depend on this category, not the other way around.